src/templates/styles.module.scss

```diff
@@ -189,4 +201,76 @@
   text-align:center;
 }
 
+/**
+ * team styling
+ */
+
+ .team_heading {
+   margin-top: 5rem;
+   display: block;
+ }
+
+ .team{
+   display:flex;
+   flex-wrap: wrap;
+   flex:60%;
+   max-width:75%;
+  }
+  .member {
+    flex: 0 0 auto;
+    min-width: 13rem;
+    margin: 3px;
+    font-size: 1rem;
+    padding: 1.5rem;
+  }
+
+  .member_name {
+    line-height: 3rem;
+  }
+
+  .twitter_button {
+    border: 2px inset $black;
+    padding: .4rem;
+  }
+
+  .twitter_icon {
+    margin-bottom: -.2rem;
+    width:1rem;
+  }
+
+  .team_avatar{
+      width: 4rem;
+      height: 4rem;
+      margin: 1em auto;
+      background-size: cover;
+      background-repeat: no-repeat;
+      background-position: center center;
+      -webkit-border-radius: 99rem;
+      -moz-border-radius: 99rem;
+      border-radius: 99rem;
+    }  
+
+  .member_details{
+    float: right;
+    margin-left: 1rem;
+  }
+
+  .screenshotsContainer{
+    display:flex;
+    flex-wrap: wrap;
+    max-width:82%;
+  }
+
+  .screenshots{
+    flex: 0 0 auto;
+    margin: 3px;
+    font-size: 1rem;
+    padding: 0.5rem;
+  }
 
+  .screenshotImage{    
+    width: 14rem;
+    height: 30rem;
+    border:1px solid;
+    box-shadow: 5px 10px 8px #888888;
+}
```

Comment:
missing new line

Reply:
I couldn't get this? it's just `box-shadow` properties.